fink_filters/filter_anomaly_notification/filter_utils.py

```diff
@@ -0,0 +1,49 @@
+import time
+import requests
+from slackclient import SlackClient
+import tokens
```

Comment:
In production, those tokens must not be loaded from a module sitting next to the code, but rather passed as runtime CLI arguments to Spark for security reasons. I propose the following. First import `os` instead of `tokens`:

```python
import os
```

Then later in the code, replace 

```python
tokens.slack_token
tokens.tg_token
```
by 
```
os.environ['ANOMALY_SLACK_TOKEN']
os.environ['ANOMALY_TG_TOKEN']
```

Then, in practice, you would launch the Spark job using:

```bash
spark-submit --master [...]
--conf spark.executorEnv.ANOMALY_SLACK_TOKEN="<your token for Slack>"\
--conf spark.executorEnv.ANOMALY_TG_TOKEN="<your token for Telegram>"\
[...]
```
This way the tokens will not have to be shipped with the code.

Reply:
Understood, thank you! Did it in the last commit